config/routes.rb

```diff
@@ -146,6 +146,13 @@
 
       scope module: :forum do
         resources :forums do
+          resources :topics do
+            post 'subscribe', on: :member
+            delete 'unsubscribe', on: :member
+            put 'locked', on: :member
+            put 'hidden', on: :member
```

Comment:
we already said that this is a property of the topic and we are putting a value?